cmd/download.go

```diff
@@ -80,8 +80,32 @@ func get(src, dest string, mode getter.ClientMode) error {
 		Mode: mode,
 	}
 	logrus.Debugf("let's get %s -> %s", src, dest)
-	err = client.Get()
+
+	gitFolder := fmt.Sprintf("%s/.git",dest)
+
+	if _, err := os.Stat(dest); !os.IsNotExist(err) {
+		logrus.Infof("%s already exists! removing it",dest)
+		removeContents(dest)
+		humanReadableDownloadLog(src, dest)
+		err = client.Get()
+		logrus.Infof("removing %s",gitFolder)
+		removeContents(gitFolder)
+	}else {
+		humanReadableDownloadLog(src, dest)
+		err = client.Get()
+		logrus.Infof("removing %s",gitFolder)
+		removeContents(gitFolder)
+	}
```

Comment:
I think we can avoid dup code using:

```go
if _, err := os.Stat(dest); !os.IsNotExist(err) {
		logrus.Infof("%s already exists! removing it",dest)
		removeContents(dest)
}
humanReadableDownloadLog(src, dest)
err = client.Get()
logrus.Infof("removing %s",gitFolder)
removeContents(gitFolder)
```

Am i right? What do you think about this approach?

Reply:
yes, this should work! Was a rusty PR i know! my intention was to do a refactoring today, but you anticipate me! I will test it with the comment removal and rename to a function! thanks @angelbarrera92 !